StarTrek13/Ships/factions.dm

```diff
@@ -204,4 +218,45 @@ var/list/global/faction_spawns = list()
 /datum/alliance/proc/removeMember(datum/faction/D)
 	if(D in member_factions)
 		member_factions -= D
-		D.broadcast("Your faction has been removed from [name]!")
\ No newline at end of file
+		D.broadcast("Your faction has been removed from [name]!")
+
+
+////objectives code\\\\
+//Shitcoded by yours truly
+
+/datum/factionobjective
+	var/iscurrent = FALSE
+	var/datum/faction/assigned_faction
+	var/description = "ERROR: THIS MESSAGE SHOULD NOT BE DISPLAYED." //The "informative" message of the objective
+
+/datum/factionobjective/New()
+	..()
```

Comment:
What’s the point of adding the child-New() if it just calls parent-New()?

Reply:
Pretty sure it's supposed to call setup() here..

Reply:
Yeah, I originally had setup() called here, move it, and forgot to delete this.

Reply:
:+1: